lib/primitives/array-element.js

```diff
@@ -92,6 +92,15 @@ module.exports = function(BaseElement, registry) {
       return undefined;
     },
 
+    /*
+     * It reduces through a given array and calls get on each item.
+     */
+    getIn: function(paths) {
```

Comment:
I could go with over loading #get to accept paths to something. You are getting an element, but just navigating a path to arrive there.